sensor_msgs/CompressedImage image
string[]               metadata_keys
string[]               metadata_values
